src/test/resources/pages/InstructorCourseDetailsWithSectionsWithHelperView.html

```diff
@@ -56,64 +56,66 @@
+                            <input name="user" value="CCDetailsUiT.instr2Helper" type="hidden">
+                            <input name="courseid" value="CCDetailsUiT.CS2103" type="hidden">
+                        </form>
+
+                            
+                            
+                        <div>
+                         <span class="help-block">
+                            Non-English characters not displayed properly in the downloaded file?<span class="btn-link" data-toggle="modal" data-target="#studentTableWindow" onclick="submitFormAjax()">
+                            click here</span>
+                         </span>
                         </div>
-                     </div>
-                  </div>
-               </div>
+                        
+                        <form id="csvToHtmlForm">
+                        <input name="courseid" value="CCDetailsUiT.CS2103" type="hidden">
+                        <input name="user" value="CCDetailsUiT.instr2Helper" type="hidden">
+                        <input name="csvtohtmltable" value="true" type="hidden">
+                        </form>
+                        
+                        <div class="modal fade" id="studentTableWindow">
+                            <div class="modal-dialog modal-lg">
+                                <div class="modal-content">
+                                    <div class="modal-header">       
+                                        
+                                         <span class="pull-left help-block">
+                                        Tips: After Selecting the table, <kbd>Ctrl + C</kbd> to COPY and <kbd>Ctrl + V</kbd> to PASTE to your Excel Workbook.
+                                        </span>
+                                        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                                        <button type="button" class="btn btn-primary" onclick="selectElementContents( document.getElementById('detailsTable') );">
+                                            Select Table</button>                                
+                                    </div>
+                                    <div class="modal-body">
+                                     <div class="table-responsive">
+                                    
+                                    <div id="detailsTable">                                  
+                                    </div>
+                                    <br>                                    
+                                    <div id="ajaxStatus">
+                                    </div>
+                                    
+                                    
+                               
+                                    </div>
+                                    </div>
+                                </div>
+                                <!-- /.modal-content -->
```

Comment:
Are you referring to the html comment? Comments are ignored when performing verifyHtml, but generating the page leaves them intact.